### DISCUSSION TITLE
Does plugin support isometric maps?

### DISCUSSION THREAD

**Discussion by neu5** [Annoraaq/grid-engine#81] [Q&A] [answered]
Does this plugin support isometric or top-down only?
It doesn't seem to work for isometric.
https://github.com/neu5-przenz-games/the-game/tree/feat/add-isometric-map
https://the-game-staging.herokuapp.com/

**Comment by Annoraaq** [comment, marked as answer]
Hi @neu5 . Currently isometric apps are not supported (if so then only by accident 😄).

It think it would be a major change. I am not saying it will never be supported but at the moment the focus lies on non-isometric maps. If non-isometric maps are the main use case (which is my assumption) then more features for non-isometric maps will be prioritized over supporting a new map type.

**Reply by Annoraaq** [reply]
But let me create an issue so we can prioritize it and don't forget it.

**Reply by Annoraaq** [reply]
@neu5 just a quick update. It might be not as hard to do and also a cool feature to have. So I think there are good chances for it to be implemented soon.

**Reply by neu5** [reply]
@Annoraaq great to hear that!
